tests/php/ModelAdminTest.php

```diff
@@ -270,4 +269,82 @@ public function testGetManagedModelTabs()
             'Tab data for managed model array using the newer syntax with dataClass and a hyphen can be generated'
         );
     }
+
+    public function testGetLinkForModelTab()
+    {
+        $admin = new ModelAdminTest\MultiModelAdmin();
+        $this->assertEquals(
+            'admin/multi/' . $this->sanitiseClassName(Contact::class),
+            $admin->getLinkForModelTab(Contact::class)
+        );
+        $this->assertEquals(
+            'admin/multi/Player',
+            $admin->getLinkForModelTab('Player')
+        );
+        $this->assertEquals(
+            'admin/multi/' . $this->sanitiseClassName(Player::class),
+            $admin->getLinkForModelTab(Player::class)
+        );
+        $this->assertEquals(
+            'admin/multi/cricket-players',
+            $admin->getLinkForModelTab('cricket-players')
```

Comment:
I'm finding this super confusing. I'm not sure what a "ModelTab" is

Based on these unit tests it can be a Model class name, or 'cricket-players' which I'm assuming is a url-segment.

I wouldn't call either of those a "tab"

Seems like the method name should be something more like `getLinkForClassNameOrModelUrlSegmentWhenUsedInModelAdmin()` (though obviously not this)

Reply:
Oh never mind

`$this->modelTab = $this->getRequest()->param('ModelClass');`

:hmm:

It's been a long day :-)